proto2ros/cmake/proto2ros_vendor_package.cmake

```diff
@@ -11,7 +11,24 @@
 #   are provided. If none is given, parent directories of PROTOS are used instead.
 # :param CONFIG_OVERLAYS: optional configuration file overlays to be applied sequentially
 #   over the default base configuration file.
-#
+# :param ROS_DEPENDENCIES: optional ROS package name to depend on for message generation and builds.
+# :param CPP_DEPENDENCIES: optional C++ targets to depend on for library builds.
+# :param CPP_INCLUDES: optional, additional C++ includes to use when building C++ sources.
+#   If nont is provided and ${CMAKE_CURRENT_SOURCE_DIR}/include/${ARG_PACKAGE_NAME} exists
```

Comment:
```suggestion
#   If none is provided and ${CMAKE_CURRENT_SOURCE_DIR}/include/${ARG_PACKAGE_NAME} exists
```

Reply:
Done in https://github.com/bdaiinstitute/ros_utilities/pull/125/commits/6afac3aad0254b1ff9bb52ab6f23451cec067ad3